rwaldron/gaia
# Bug 950914 - [Messages][Drafts] Replace existing draft when hiding app while showing unsent, existing draft content

- `message_manager.js`
  - specify that the draft should be preserved when app hides
- `thread_ui.js`
  - if draft is specified to be preserved in `saveDraft`, preserve it

Tests
- `message_manager_test.js`
  - make sure preserve flag is passed when app hides
- `thread_ui_test.js`
  - make sure draft is preserved (or not) depending on what is specified in `saveDraft`
